src/current/_includes/v24.1/known-limitations/schema-changes-between-prepared-statements.md

```diff
@@ -1,5 +1,7 @@
 When the schema of a table targeted by a prepared statement changes before the prepared statement is executed, CockroachDB allows the prepared statement to return results based on the changed table schema, for example:
 
+When object definitions referenced in a prepared statement are altered the prepared statement is created, future executions of the prepared statement could result in an error. For example, adding a column to a table referenced in a prepared statement with a `SELECT *` clause will result in an error:
```

Comment:
It looks like there is a missing word. Is this correct?

```suggestion
When object definitions referenced in a prepared statement are altered the after prepared statement is created, future executions of the prepared statement could result in an error. For example, adding a column to a table referenced in a prepared statement with a `SELECT *` clause will result in an error:
```

Reply:
Good catch. LGTM